wazimap_ng/general/views.py

```diff
@@ -82,6 +82,24 @@ def indicator_data_for_children(request, profile_id, geography_code):
     )
     return Response(profile_js)
 
+@condition(etag_func=etag_profile_updated, last_modified_func=last_modified_profile_updated)
+@api_view()
+def consolidated_profile_summary(request, profile_id, geography_code):
+    version_name = request.GET.get('version', None)
+    profile = get_object_or_404(profile_models.Profile, pk=profile_id)
+    if version_name is None:
+        version_name = profile.geography_hierarchy.default_version
```

Comment:
Maybe it would be better to combine the version_name assignments like 

`version_name = request.GET.get('version', profile.geography_hierarchy.default_version)`

Reply:
updated